SwiftBasicsExercise.playground/Pages/ControlFlow_5.1.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,53 @@
+import Foundation
+//Control-flow exercise 1
+print("\n1-->")
+for i in stride(from: 0, to: 100,by: 1) {
+    let power = Int(pow(Float(2), Float(i)))
+    if power > 100 {
+        break
+    } else {
+        print("\(power) ", terminator: "")
+    }
+}
```

Comment:
This is a lot of work here,
we can simply use while loop
let N = 100
var ans = 2

while(ans <= N) {
print(ans)
ans *= 2
}